### PR TITLE
add piechart in account detail

### PR DESCRIPTION
**Pull request by hongdnn**
Capture

### CODE REVIEW

File: lib/app/definition/app_assets.dart [hunk 1]
```diff
@@ -33,6 +33,10 @@ class AppAssets {
   static final String icDig = _generateImagePath('ic_dig.png');
   static final String icArrowUp = _generateImagePath('ic_arrow_up.png');
   static final String icArrowDown = _generateImagePath('ic_arrow_down.png');
+  static final String icBalance = _generateImagePath('ic_balance.png');
```
Comment (manhduc-nguyen) [inline lib/app/definition/app_assets.dart]: Screen Shot 2022-06-23 at 10 30 49
lets use svg image if possible, less resources size

File: lib/app/page/active_account/active_account_detail/active_account_detail_page.dart [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+  @override
+  Widget build(BuildContext context) {
+    final List<PieChartSeries> data = [
+      PieChartSeries('Balance', 0.58, 2637.83, 0, const Color.fromRGBO(42, 157, 143, 1), AppAssets.icBalance),
+      PieChartSeries('Delegation', 99.33, 4502, 0, const Color.fromRGBO(156, 183, 211, 1), AppAssets.icDelegation),
+      PieChartSeries('Reward', 0.58, 374.79, 0, const Color.fromRGBO(233, 196, 106, 1), AppAssets.icReward),
+      PieChartSeries('Unbonding', 0.0, 100, 0, const Color.fromRGBO(231, 111, 81, 1), AppAssets.icUnbonding),
+    ];
+
+    // final List<charts.Series<PieChartSeries, int>> series = [
```
Comment (manhduc-nguyen) [inline lib/app/page/active_account/active_account_detail/active_account_detail_page.dart]: can u remove unused code pls

**Comment by manhduc-nguyen** [comment]
@hongdnn also fix the github CI failed pls

**Comment by hongdnn** [comment]
Thanks, I fixed it.

**Review by manhduc-nguyen — APPROVED** [review]
LGTM

**Pull request merged**